src/wdio/services/AxeService.js

```diff
@@ -0,0 +1,47 @@
+/* global browser, axe */
+import fs from 'fs';
+
+let axeCoreSrc = fs.readFileSync('node_modules/axe-core/axe.min.js', 'utf8');
+axeCoreSrc = axeCoreSrc.replace(/^\/\*.*\*\//, '');
```

Comment:
Would need to test this with windows

Reply:
why would this change on windows? ALl its doing is stripping out leading comments in the JS so it injects nice